build.gradle

```diff
@@ -5,12 +5,11 @@ plugins {
   id 'maven-publish'
   id 'signing'
   id 'net.researchgate.release' version '2.8.1'
-  id 'io.codearte.nexus-staging' version '0.21.2'
-  id 'com.gradle.plugin-publish' version '0.11.0'
-  id 'com.github.johnrengelman.shadow' version '5.2.0'
+  id 'io.codearte.nexus-staging' version '0.30.0'
+  id 'com.gradle.plugin-publish' version '0.14.0'
+  id 'com.github.johnrengelman.shadow' version '6.1.0'
```

Comment:
Was hoping this might address issue with [IntelliJ](https://github.com/johnrengelman/shadow/issues/264)  jars but no luck :disappointed: